ci/gitlab/.gitlab-ci.yml

```diff
@@ -293,6 +293,7 @@ Ptest qemux86_64:
       GIT_CHECKOUT: false
   dependencies:
     - OE Checkout
+  needs: ["OE Checkout"]
```

Comment:
Why does this one need both keywords specified?

Reply:
Pre-lunch commits are rarely good idea :-/